docs/en/get_started/installation.md

```diff
@@ -2,183 +2,374 @@
 ```
 
-For more details, please refer the the following tables and delete `=={mmcv_version}`.
+If you find that the above installation command does not use a pre-built package ending with `.whl` but a source package ending with `.tar.gz`, you may not have a pre-build package corresponding to the PyTorch or CUDA or mmcv-full version, in which case you can [build mmcv-full from source](https://mmcv.readthedocs.io/en/latest/get_started/build.html).
+
+<details>
+<summary>Installation log using pre-built packages</summary>
+
+Looking in links: https://download.openmmlab.com/mmcv/dist/cu102/torch1.8.0/index.html<br />
+Collecting mmcv-full<br />
+<b>Downloading https://download.openmmlab.com/mmcv/dist/cu102/torch1.8.0/mmcv_full-1.6.1-cp38-cp38-manylinux1_x86_64.whl</b>
+
+</details>
+
+<details>
+<summary>Installation log using source packages</summary>
+
+Looking in links: https://download.openmmlab.com/mmcv/dist/cu102/torch1.8.0/index.html<br />
+Collecting mmcv-full==1.6.0<br />
+<b>Downloading mmcv-full-1.6.0.tar.gz</b>
 
-ii. Install a specified version.
+</details>
 
-The rule for installing a specified `mmcv-full` is as follows:
+To install a specific version of mmcv-full, for example, mmcv-full version 1.6.0, you can use the following command
 
-```shell
-pip install mmcv-full=={mmcv_version} -f https://download.openmmlab.com/mmcv/dist/{cu_version}/{torch_version}/index.html
+```bash
+mim install mmcv-full==1.6.0
 ```
 
-First of all, please refer to the Releases and replace `{mmcv_version}` a specified one. e.g. `1.3.9`.
-Then replace `{cu_version}` and `{torch_version}` in the url to your desired versions. For example,
-to install `mmcv-full==1.3.9` with `CUDA 11.1` and `PyTorch 1.9.0`, use the following command:
+:::{note}
+If you would like to use `opencv-python-headless` instead of `opencv-python`,
+e.g., in a minimum container environment or servers without GUI,
+you can first install it before installing MMCV to skip the installation of `opencv-python`.
 
-```shell
-pip install mmcv-full==1.3.9 -f https://download.openmmlab.com/mmcv/dist/cu111/torch1.9.0/index.html
+Alternatively, if it takes too long to install a dependency library, you can specify the pypi source
+
+```bash
+mim install mmcv-full -i https://pypi.tuna.tsinghua.edu.cn/simple
 ```
 
-```{note}
+:::
+
+You can run [check_installation.py](https://github.com/open-mmlab/mmcv/.dev_scripts/check_installation.py) to check the installation of mmcv-full after running the installation commands.
+
+#### Install with pip
+
+Use the following command to check the version of CUDA and PyTorch
+
+```bash
+python -c 'import torch;print(torch.__version__);print(torch.version.cuda)'
+```
+
+Select the appropriate installation command depending on the type of system, CUDA version, PyTorch version, and MMCV version
+
+<html>
+<body>
+    <style>
+      select {
+          z-index: 1000;
+          position: absolute;
+          top: 10px;
+          width: 6.7rem;
+      }
```

Comment:
The following html block seems messed up on Github page. Is it ok after sphinx build?

Reply:
Sphinx builds it as expected.
![image](https://user-images.githubusercontent.com/58739961/196110516-5e015675-5f6a-4f3c-a311-07f3f0380609.png)